<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToe</class>
 <widget class="QDialog" name="TicTacToe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>388</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>656</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(42, 42, 41);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_gameName">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 22px;
color:white
</string>
       </property>
       <property name="text">
        <string>TIC-TAC-TOE</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_whichTurn">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: #1e1e1e;
    border: 2px solid #555;
    border-radius: 10px;
    font-size: 20px;
    font-weight: bold;
    color: #e0e0e0;
    padding: 10px;
    min-width: 100px;
    min-height: 30px;
    text-align: center;
}

QLabel:disabled {
    background-color: #4a4a4a;
    color: #888;
}</string>
       </property>
       <property name="text">
        <string>YOUR TURN</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_x">
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
      background-color:  #586375;
      border: 1px solid #3B414D;
      font-size: 24px;
      font-weight: bold;
      padding: 6px 6px;
      min-width: 40px;
	 border-top-left-radius: 10px;
      border-bottom-left-radius: 10px;
      border-right: none; 
    
    }
    QPushButton:last-child {
      border-top-right-radius: 10px;
      border-bottom-right-radius: 10px;
      border-left: none; 
    }

    QPushButton:checked {
      background-color:  rgb(40, 44, 52);
      border: 1px solid #3B414D;
    }

    QPushButton:hover {
      background-color: #4C5566;
    }

    QPushButton:pressed {
      background-color: #586375;
      border: 1px solid #3B414D;
    }

    QHBoxLayout {
      margin: 0;
    }</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_0">
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
      background-color:  #586375;
      border: 1px solid #3B414D;
      font-size: 24px;
      font-weight: bold;
      min-width: 40px;
padding: 6px 6px;
	 border-top-right-radius: 10px;
      border-bottom-right-radius: 10px;
      border-left: none; 
    
    }
    QPushButton:last-child {
      border-top-right-radius: 10px;
      border-bottom-right-radius: 10px;
      border-left: none; 
    }

    QPushButton:checked {
      background-color:  rgb(40, 44, 52);
      border: 1px solid #3B414D;
    }

    QPushButton:hover {
      background-color: #4C5566;
    }

    QPushButton:pressed {
      background-color: #586375;
      border: 1px solid #3B414D;
    }

    QHBoxLayout {
      margin: 0;
    }</string>
         </property>
         <property name="text">
          <string>O</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #000000;
  border: 2px solid #1A1A1A;
  border-radius: 15px;
  color: #FFFFFF;
  font-size: 14px;
  font-weight: 600;
  margin: 0;
  min-height: 20px;
  padding: 16px 24px;
  text-align: center;
  transition: all 300ms cubic-bezier(.23, 1, 0.32, 1);
  width: 100%;
}

QPushButton:hover {
  background-color: #333333;
  border: 2px solid #555555;
  box-shadow: 0 8px 15px rgba(0, 0, 0, 0.25);
  margin: 0 0 6px 0;
}

QPushButton:pressed {
  background-color: #1A1A1A;
  border: 2px solid #333333;
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.5);
  margin: 0 0 6px 0;
}

QPushButton {
  outline: none; 
}</string>
       </property>
       <property name="text">
        <string>START!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #000000;
  border: 2px solid #1A1A1A;
  border-radius: 15px;
  color: #FFFFFF;
  font-size: 14px;
  font-weight: 600;
  margin: 0;
  min-height: 20px;
  padding: 16px 24px;
  text-align: center;
  transition: all 300ms cubic-bezier(.23, 1, 0.32, 1);
  width: 100%;
}

QPushButton:hover {
  background-color: #333333;
  border: 2px solid #555555;
  box-shadow: 0 8px 15px rgba(0, 0, 0, 0.25);
  margin: 0 0 6px 0;
}

QPushButton:pressed {
  background-color: #1A1A1A;
  border: 2px solid #333333;
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.5);
  margin: 0 0 6px 0;
}

QPushButton {
  outline: none; 
}</string>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
     min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>84</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2e2e2e;
  
 border: 2px solid #555;
    font-size: 28px;
    font-weight: bold;
    color: #e0e0e0;
    min-width: 100px;
    min-height: 80px;
}

QPushButton:hover {
    background-color: #3a3a3a;
}

QPushButton:pressed {
    background-color: #1f1f1f;
    border: 2px solid #777;
}

QPushButton:disabled {
	color:  rgb(235, 235, 235); 
    background-color: #4a4a4a;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
